resources/destination-types/googlecloud.yaml

```
@@ -189,6 +189,31 @@ spec:
         sectionHeader: true
         gridColumns: 12
       advancedConfig: true
+    
+    - name: enable_wal
+      label: Metric Write-Ahead-Log
+      description: |
+        Write metrics to a write-ahead-log before sending to help ensure
+        metrics are not lost in case of network outages or collector restarts.
+      type: bool
+      default: false
+      options:
+        sectionHeader: true
+        gridColumns: 12
+      advancedConfig: true
+    
+    - name: wal_max_backoff
+      label: WAL Max Retry Duration
+      description: The maximum amount of time in minutes to retry sending metrics from the write-ahead-log when a network error occurs.
```

```suggestion
      description: The maximum amount of time (minutes) to retry sending metrics from the write-ahead-log when a network error occurs.
```
Nitpick, just to be consistent with our collection interval descriptions